src/GithubMergeTool/GithubMergeTool.cs

```diff
@@ -187,32 +197,37 @@ git commit
 ```
 Once all conflicts are resolved and all the tests pass, you are free to merge the pull request.";
 
-            prMessage = prMessage.Replace(newLine, "\\n");
+            Console.WriteLine("Creating PR");
 
             // Create a PR from the new branch to the dest
-            body = $@"
-{{
-    ""title"": ""{prTitle}"",
-    ""body"": ""{prMessage}"",
-    ""head"": ""{prBranchName}"",
-    ""base"": ""{destBranch}""
-}}";
-
-            Console.WriteLine("Creating PR");
-            response = await _client.PostAsyncAsJson($"repos/{repoOwner}/{repoName}/pulls", body);
+            // https://developer.github.com/v3/pulls/#create-a-pull-request
+            response = await _client.PostAsyncAsJson($"repos/{repoOwner}/{repoName}/pulls", JsonConvert.SerializeObject(
+                new
+                {
+                    title = prTitle,
+                    body = prMessage,
+                    head = prBranchName,
+                    @base = destBranch
```

Comment:
what does the @ do/mean?

Reply:
base is a reserved word, so to name something base you need the `@`